packages/types/src/ApiActivity.ts

```diff
@@ -0,0 +1,29 @@
+import z from 'zod'
+
+import { branded } from './branded'
+import { UnixTime } from './UnixTime'
+
+const ChartPoint = z.tuple([
+  branded(z.number(), (n) => new UnixTime(n)),
+  z.number().int(),
+])
+export type ChartPoint = z.infer<typeof ChartPoint>
+
+const Chart = z.object({
+  types: z.tuple([z.literal('timestamp'), z.string()]),
+  data: z.array(ChartPoint),
+})
+export type Chart = z.infer<typeof Chart>
+
+export const Charts = z.object({
+  hourly: Chart,
+  sixHourly: Chart,
+  daily: Chart,
+})
+export type Charts = z.infer<typeof Charts>
+
+export const ApiActivity = z.object({
+  combined: Charts,
```

Comment:
Do we actually need combined data here? If I understand it right it's easily derived from project charts. If that's the case, then by removing combined data we could speed up the api http response and lower its memory footprint. Wdyt? cc @sz-piotr @krzkaczor

Reply:
I would keep the combined data here, because the fact that we're calculating it in the app layer is an implementation detail, we might have as well had it calculated in a query, it just isn't what we decided on

Reply:
That's a good point. At the end of the day it's perf vs dx trade-off - if we find it too slow/memory-intensive nothing stops us from moving calculation to the frontend - for now let's keep it here.